Doc/constraints.txt

```diff
@@ -0,0 +1,17 @@
+babel<3
+colorama<0.5
+imagesize<1.5
+Jinja2<3.2
+MarkupSafe<2.2
+packaging<24
+polib<1.3
+Pygments<2.16,!=2.15.0  # Pygments==2.15.0 breaks CI
```

Comment:
I'd add the comment on the preceding line, as I forsee more exclusions being added and I think this would make it easier to read (also fits with my general suggestion above)

```suggestion
# Pygments==2.15.0 breaks CI
Pygments<2.16,!=2.15.0
```